hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/autocleanup/AutoActionCleanupTest.java

```diff
@@ -97,7 +97,7 @@ public void cleanupDisabled() {
 
     @Test
     @Description("Verifies that canceled and failed actions are cleaned up.")
-    public void canceledAndFailedActionsAreCleanedUp() {
+    public void canceledAndFailedActionsAreCleanedUp() throws Exception{
```

Comment:
Superfluous declaration of exception 
```suggestion
    public void canceledAndFailedActionsAreCleanedUp(){
```